### ISSUE TITLE
Can you please add a price per coin view in account view?

### ISSUE THREAD

**Issue by staticfrost** [LedgerHQ/ledger-live-desktop#1614]
Hi,

Its great that I can see my total coins worth, but i'd like to see a break down based on per coin?

**Comment by meriadec** [comment]
Hi, legitimate request but currently blocked by design

closing in favor of https://github.com/LedgerHQ/ledger-live-desktop/issues/984

**Comment by juan-cortes** [comment]
I'm surprised we don't have that piece of information available in the UI, to be honest. @meriadec couldn't we do something like show the price per coin alongside the total? I think @staticfrost is referring to having the current price of the coin in the account view page, something like this mockup

![image](https://user-images.githubusercontent.com/4631227/47711545-c584eb00-dc35-11e8-9aae-5ad1f1489118.png)

**Comment by gre** [comment]
i also myself thought it was mentioning another thing.

I think the relevant discussion on this was #1189

**Comment by meriadec** [comment]
Oops sorry I read it too fast :man_facepalming:
